velox/functions/prestosql/types/tests/HyperLogLogTypeTest.cpp

```diff
@@ -25,6 +25,15 @@ class HyperLogLogTypeTest : public testing::Test, public TypeTestBase {
   }
 };
 
+TEST_F(HyperLogLogTypeTest, basic) {
+  ASSERT_EQ(HYPERLOGLOG()->name(), "HYPERLOGLOG");
```

Comment:
is `toString()` method already tested somewhere else?

Reply:
@pedroerp I don't think. Should I add some tests?

Reply:
I would say so, for completeness.